lib/evmone/instructions_traits.hpp

```diff
@@ -200,6 +200,9 @@ constexpr inline GasCostTable gas_costs = []() noexcept {
 static_assert(gas_costs[EVMC_MAX_REVISION][OP_ADD] > 0, "gas costs missing for a revision");
 
 
+constexpr auto REV_EOF1 = EVMC_PRAGUE;
```

Comment:
Good idea?

Reply:
Is the name good? For `TXCREATE` we need something like "EOF v1.1".

Reply:
I think it's okay. Also okay to drop `TXCREATE`